ci/install.sh

```diff
@@ -45,11 +45,23 @@ EOF
     fi
 }
 
+install_zsh() {
+    case $TARGET in
+        aarch64-unknown-linux-gnu)
```

Comment:
Ah, I [see now](https://github.com/BurntSushi/ripgrep/blob/master/ci/install.sh#L9). Yeah, all boiler plate. The original source is [rust-everywhere](https://github.com/japaric/rust-everywhere).